pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUmbrellaStudy.java

```diff
@@ -43,7 +43,7 @@ public interface JdbiUmbrellaStudy extends SqlObject {
 
     @SqlQuery("select us.umbrella_study_id, us.umbrella_id, us.study_name, us.guid, us.irb_password, us.web_base_url, us.auth0_tenant_id,"
             + " op.olc_precision_code, us.share_participant_location, us.study_email, us.enable_data_export, us.recaptcha_site_key,"
-            + " us.default_auth0_connection, us.error_present_status_enabled"
+            + " us.default_auth0_connection, us.error_present_status_enabled, us.notification_email"
```

Comment:
probably insert should also be updated to handle this new column !

Reply:
The column is nullable with a default null value. So, no need to update the insert statement.